decidim-verifications/.rubocop_rails.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Why does this single folder need all these contents?

Reply:
Because my bash commands to batch-create symlinks have bugs :joy:. Let me fix it.

Reply:
😂

Reply:
Amended, should be fine now I think.